plugin.yml

```diff
@@ -1,45 +1,23 @@
----
-#     _    __                  _                                     _
-#    | |  / /                 | |                                   | |
-#    | | / /                  | |                                   | |
-#    | |/ / _   _  ____   ____| | ______ ____   _____ ______   ____ | | __
-#    | |\ \| | | |/ __ \ / __ \ |/ /  __/ __ \ / __  | _  _ \ / __ \| |/ /
-#    | | \ \ \_| | <__> |  ___/   <| / | <__> | <__| | |\ |\ | <__> |   <
-# By |_|  \_\__  |\___  |\____|_|\_\_|  \____^_\___  |_||_||_|\____^_\|\_\
-#              | |    | |                          | |
-#           ___/ | ___/ |                          | |
-#          |____/ |____/                           |_|
-#
-# A PocketMine-MP plugin to see the server TPS and a player's ping
-# Copyright (C) 2020-2021 Kygekraqmak
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# he Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-
+ # See your and others's ping and the server tps
+ # Copyright © 2021 KygekTeam
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
 name: KygekPingTPS
-version: 1.0.1
-main: Kygekraqmak\KygekPingTPS\Main
+config: "1.1"
+version: 1.0.2
```

Comment:
You could change this to version 1.1.0 because it's a minor update, not a patch update.